src/NzbDrone.Core/Parser/QualityParser.cs

```diff
@@ -73,7 +73,7 @@ public class QualityParser
 
         private static readonly Regex HighDefPdtvRegex = new Regex(@"hr[-_. ]ws", RegexOptions.Compiled | RegexOptions.IgnoreCase);
 
-        private static readonly Regex RemuxRegex = new Regex(@"\b(?<remux>(BD|UHD)?[-_. ]?Remux)\b", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+        private static readonly Regex RemuxRegex = new Regex(@"[_. ](?<remux>(?:(BD|UHD)[-_. ])?Remux)\b", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
```suggestion
        private static readonly Regex RemuxRegex = new Regex(@"[_. ](?<remux>(?:(BD|UHD)[-_. ]?)?Remux)\b", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```
should fix the testcase that failed